dali/operators/input/video_input.h

```diff
@@ -112,7 +170,12 @@ class VideoInput : public VideoDecoderBase<Backend, FramesDecoder>, public Input
 
   bool SetupImpl(std::vector<OutputDesc> &output_desc, const Workspace &ws) override;
 
-  void RunImpl(Workspace &ws) override;
+  /**
+   * Awkward RunImpl function for VideoInput operator.
+   * @see VideoInputCpu & VideoInputMixed for more information.
+   */
+  void ViRunImpl(Workspace &ws);
```

Comment:
nitpick: The `Vi` prefix doesn't tell me much.

Reply:
Changed name to `VideoInputRunImpl`